tests/serenity/src/test/java/bla/tm/definitions/site/pantheon/UserLogInDefinition.java

```diff
@@ -24,10 +26,20 @@ public void openCreateNewAccountPage() {
     @Given("navigate to Pantheon User Account page from LogIn page")
     @When("navigate to Pantheon User Account page from LogIn page")
     public void openUserAccountPage(){
-        userLogInPage.logInToAccount();
+        userLogInPage.logInToApp(admin.username, admin.password);
+    }
+
+    @When("login to Pantheon with $username and $password")
+    public void logInToApp(String username, String password){
```

Comment:
Not sure I understand a logic
why do you need to create UserDate obj with hard coded login and password and then get them from BDD scenario?